app/models/user.rb

```diff
@@ -2,4 +2,5 @@ class User < ApplicationRecord
   include Clearance::User
 
   has_many :poems, dependent: :destroy
```

Comment:
I don't think so here. The user could feasibly create a poem and track it here before it has any associated submissions.